app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+
+const app = express();
+
+app.set('view engine', 'hbs');
+app.set('views', `${__dirname}/views`)
+
+app.get('/', (req, res, next) => {
+    res.render('home', {})
+})
+
+app.get('/about', (req, res, next) => {
+    res.render('about', {})
+})
+
+app.get('/works', (req, res, next) => {
+    res.render('works', {})
+})
+
+app.get('/gallery', (req, res, next) => {
+    res.render('gallery', {})
+})
```

Comment:
```suggestion
app.get('/', (req, res, next) => {
    res.render('home')
})

app.get('/about', (req, res, next) => {
    res.render('about')
})

app.get('/works', (req, res, next) => {
    res.render('works')
})

app.get('/gallery', (req, res, next) => {
    res.render('gallery')
})
```

Entiendo porque lo has puesto pero en este caso no te harian falta, ya que no les esta pasando ninguna variable, mejor no lo ponemos y asi queda mas limpio el codigo :D, ahora bien si le vas a pasar a la vista alguna variable entonces si lo ponemos :D